examples/tls/requirements.txt

```diff
@@ -1,4 +0,0 @@
-grpcio==1.38.1
```

Comment:
same here

Reply:
this file has been removed and readme updated